src/main/java/org/folio/orders/utils/validators/CompositePoLineValidationUtil.java

```diff
@@ -252,4 +255,15 @@ private static Error convertErrorCodesToError(CompositePoLine compPOL, ErrorCode
     }
     return error;
   }
+
+  private static List<ErrorCodes> validatePackagePoLine(CompositePoLine compPOL) {
+     List<ErrorCodes> errors = new ArrayList<>();
+
+     // The quantity of the electronic resources in the cost must be specified
+     if (compPOL.getIsPackage() && compPOL.getInstanceId() != null) {
```

Comment:
Sonarcloud is complaining about `getIsPackage()` for NPE. Think it wants  `Boolean.TRUE.equals(compPOL.getIsPackage())` to safeguard against NPE.

Reply:
updated